packages/vscode-extension/messages.json

```diff
@@ -5,6 +5,7 @@
 	"COMMAND_SUCCESS": "The {0} command has been successfully executed.",
 	"COMMAND_TRIGGERED": "{0} has been triggered.",
 	"DISMISS": "Dismiss",
+	"PRODUCTION_DOMAIN": "system.netsuite.com",
```

Comment:
You can reuse the ApplicationConstants.PROD_ENVIRONMENT_ADDRESS from packages/nodecli. In vscode, ApplicationConstants is accessed via ExtensionUtil.